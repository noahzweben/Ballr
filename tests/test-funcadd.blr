func int sum(int x, int y) {
  return x + y;
}


gameboard g1{
	clr = (251,142,74);
	size = (680,420);
	init -> {
		int result = sum(5,4);
		print(result); 
	}
}
