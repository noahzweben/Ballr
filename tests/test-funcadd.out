9
SDL could not initialize! SDL Error: No available video device
Window initialization failed.